public-config.yml

```diff
@@ -6,7 +6,6 @@ buildpack-packager-git-uri-public: https://github.com/cloudfoundry/buildpack-pac
 buildpack-packager-git-uri: git@github.com:cloudfoundry/buildpack-packager.git
 buildpack-release-candidates-bucket: buildpack-release-candidates
 buildpackapplifecycle-git-branch: develop
-buildpacks-binaries-s3-bucket: pivotal-buildpacks
```

Comment:
This is removed because it was only used in one place (and hence inlined in that location), and I think the indirection is not helpful.